cpp/dolfinx/la/PETScKrylovSolver.h

```diff
@@ -35,16 +35,16 @@ class PETScKrylovSolver
   /// @param[in] inc_ref_count Increment the reference count on `ksp` if true
   PETScKrylovSolver(KSP ksp, bool inc_ref_count);
 
-  // Delete copy constructor
```

Comment:
The deleted lines are better - we don't need to document something that is deleted.